### PR TITLE
QDomyos-Zwift WebSocket API Installation & Operation guide

### PR DESCRIPTION
**Pull request by gungun974**
As promised in issue #572 here is a complete tutorial on how to install QDomyos-Zwift on a raspberry Pi / Linux but also quickly understand the base API.  (Okay I'm late of one day :sweat_smile:)

(I didn't write all the commands and I don't know how in markdown to ref to another file :eyes:)

Sorry if there are a lot of spelling or grammar mistakes but English is not my mother language and already I have big spelling and grammar problems in French xD (I'm more a guy who program than write docs xD)

### CODE REVIEW

**Comment by cagnulein** [comment]
Thank you!

**Comment by satchafunkilus** [comment]
Somehow, I am having quite some issues getting this to work. I would like to access the QZ data from my home automation system in order to control a fan. However, it's not really clear for me on how to achieve this. Since I did not manage to get the httpserver to work on linux, I have tried this with the windows version that already ships with the server. What confuses me: 

1. QZ seems to spin up the http server on a random port every time it is started. I can eventually find out the port from the console output and have found some webinterface on http://ip:random_port/floating/floating.htm. but this is not reproducible, since the port keeps changing.
  - Is there any way to fix this port number? Why does this keep changing?
  - When I enable the user_QZWS webserver and try to connect to that port, I get a message saying the site has a temporary problem or has moved. 

So long story short - what am I missing or doing wrong here? What would be the easiest way to get the data from QZ into nodered, homeassistant, etc.?

**Comment by cagnulein** [comment]
For the web server static port there is already a ticket about it, I will try to do ASAP https://github.com/cagnulein/qdomyos-zwift/issues/2144

For the qzws the expert is @p3g4asus  :) 

I also added osc in this branch, probably it could be your best shot https://github.com/cagnulein/qdomyos-zwift/pull/2449

I'm on a business travel this week I will try to support you ASAP
